src/core/storeData.ts

```diff
@@ -1,15 +1,12 @@
 import { Base64 } from "js-base64";
 import { logger } from ".";
-import { EnhancedDataEvent } from "../interfaces";
+import { MetricData } from "../interfaces";
 import octokit from "./octokit";
 
-export const storeData = async (
-  enhancedDataEvents: (EnhancedDataEvent | Promise<EnhancedDataEvent>)[]
-) => {
-  if (!enhancedDataEvents) return false;
-  const resolvedEnhancedDataEvents = await Promise.all(enhancedDataEvents);
```

Comment:
I only could have a quick look but it is right that we don't have promises here anymore right?

Reply:
Yes. We pass the `await`ed result of `collectMetrics` function to `storeData`, which is an array of `MetricData`.